source/elements/oneMKL/source/domains/rng/device_routines/device-distributions-template-parameter-method.rst

```diff
@@ -0,0 +1,49 @@
+.. SPDX-FileCopyrightText: 2023 Intel Corporation
+..
+.. SPDX-License-Identifier: CC-BY-4.0
+
+.. _onemkl_device_rng_uniform_continuous_distributions_method:
+
+Distributions Template Parameter Method
+=======================================
+
+.. tabularcolumns:: |\Y{0.4}|\Y{0.2}|\Y{0.4}|
+
+.. list-table::
+   :header-rows: 1
+   :class: longtable
+
+   * -  Method Type
+     -  Distributions
+     -  Math Description
+
+   * -  ``uniform_method::standard``
+        ``uniform_method::accurate``
+     -  ``uniform``
+     -   Standard method. Currently there is only one method for these functions. ``uniform_method::accurate`` checks for additional ``s`` and ``d`` data types. For ``integer`` data types, it uses ``d`` as a ``BRNG`` data type (``s`` ``BRNG`` data type is used in ``uniform_method::standard`` method on GPU).
```

Comment:
"Standard method. Currently there is only one method for these functions." - looks like there are two methods "standard" and "accurate"

Reply:
removed, thanks